README.md

```diff
@@ -17,9 +17,9 @@ Set of standardized Web Content Management (WCM) components for [Adobe Experienc
 * **[Component Library](https://www.adobe.com/go/aem_cmp_library):** A collection of examples to view the components in their various configurations.
 * **[Component Documentation](https://docs.adobe.com/content/help/en/experience-manager-core-components/using/introduction.html):** For developers and authors, with details about each component.
 * Get Started:
```

Comment:
Unrelated spaces, may be better to remove this change.

Reply:
Done.